### ISSUE TITLE
Comma after ellipsis

### ISSUE THREAD

**Issue by moewew** [plk/biblatex-apa#46]
Consider
```latex
\documentclass[paper=a4]{article}
\usepackage[american]{babel}
\usepackage{csquotes}

\usepackage[style=apa]{biblatex}

\usepackage{filecontents}
\begin{filecontents}{\jobname.bib}
@ARTICLE{ao1,
  AUTHOR         = {Boker, S. and Neale, M. and Maes, H. and Wilde, M. and
                   Spiegel, M. and Brick, T. and Spies, J. and Estabrook,
                   R. and Kenny, S. and Bates, T. and others},
  TITLE          = {Open{M}x: {A}n open source extended structural
                   equation modeling framework},
  JOURNAL        = {Psychometrika},
  VOLUME         = {76},
  NUMBER         = {2},
  PAGES          = {306--317},
  DATE           = {2011}
}
\end{filecontents}
\addbibresource{\jobname.bib}

\begin{document}
\cite{ao1}
\printbibliography
\end{document}
```
gives
> Brick, T., … Bates, T.

for me with `biblatex` 3.10.

`reference.pdf` has

> Brick, T., …, Bates, T.

at that point.

Note that with pdfLaTeX the undesired behaviour was present even before 3.10. This is due to the harmonisation of the ellipsis in https://github.com/plk/biblatex/commit/5de1c77d639e4fdea5506bfdb3c90ec63be8c35c following https://github.com/plk/biblatex/issues/514. 

PS: Should 'United States' and 'United Kingdom' not be protected against case changes in the `.bib` files?

**Comment by plk** [comment]
Can you tell me which entry in the references .bib has the comma? I can check the APA manual.

**Comment by moewew** [comment]
* `7.01:2` (Gilbert, ... Botros) has no comma.
* `ao1` (Boker, ..., Bates) is a citation with comma
* `weissetal2009a` (Weiss, ... King) is a citation without comma

**Comment by moewew** [comment]
OK. I guess this has become obsolete and I was working under the wrong assumptions. With 3.10 there are no commas after the ellipsis and apparently this is what APA style dictates.

https://github.com/plk/biblatex/issues/694 and https://github.com/plk/biblatex/pull/704 will require `\newcommand*{\begrelateddelimreviewof}{\addspace}` for `biblatex` 3.11 to give the expected output for `review` items. (I can submit a PR if you like) Other than that you should be fine moving `biblatex-apa` from 3.10 to 3.11.

**Comment by plk** [comment]
I have fixed it all for 3.11, thanks.
